pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -16,8 +17,23 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+type ExternalResources []struct {
+	Kind           string            `json:"kind"`
+	Data           map[string]string `json:"data,omitempty"`
+	Name           string            `json:"name"`
+	PoolName       string            `json:"pool-name,omitempty"`
+	FilesystemName string            `json:"filesystem-name,omitempty"`
+}
+
+const (
+	secretName      = "rook-ceph-external-cluster-details"
+	secretNamespace = "openshift-storage"
+	clusterDetails  = "external_cluster_details"
+)
+
 // ensureStorageClasses ensures that StorageClass resources exist in the desired
 // state.
+
```

Comment:
unnecessary empty line

Reply:
My vscode apparently don't look for empty lines at times, Fixed it, thanks